pages/admin/login.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState, useEffect } from 'react';
+import jwt_decode from "jwt-decode";
+const LitJsSdk = require('lit-js-sdk');
+import { Spinner } from '@/common/components/elements/Spinner';
+
+const accessControlConditions = [
```

Comment:
Can you add a type declaration to this? Especially given that [getSignedToken accepts a plain js object](https://lit-protocol.github.io/lit-js-sdk/api_docs_html/#litnodeclientgetsignedtoken), it's not clear what's required.